File: Java/com/manish/assignments/eleven/Employee.java
package com.manish.assignments.eleven;

public class Employee {
	private String fistName;
	private String lastName;
	
	private double salary;
	
	Employee(String fName, String lName){
		this.fistName=fName;
		this.lastName=lName;
	}
	
	public void setSalary(double m) {this.salary=m;}
	public double getSalary() {return this.salary;}
	
	public String getName() {return this.fistName+this.lastName;}
}

File: CPP/11/src/Employee.cpp
/*
 * Emloyee.cpp
 *
 *  Created on: 17-Aug-2024
 *      Author: Manish patil
 */

#include "Employee.h"

Employee newEmployee(string fName,string lName, double salary){
	Employee t;
	t.fName=fName;
	t.lName=lName;
	t.salary=salary;
	return t;
}

void setSalary(Employee* e,double salary){
	e->salary=salary;
}
void empDisplay(Employee* e){
	cout<<"Employee "<<e->fName<<" "<<e->lName<<" has salary "<<e->salary;
}
